app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -35,10 +37,18 @@
     ProgressBar progressBar;
     @BindView(R.id.toolbar)
     Toolbar toolbar;
+    @BindView(R.id.checkboxEnableUrl)
+    CheckBox checkBoxEnableUrl;
+    @BindView(R.id.addUrlContainer)
+    View addUrlContainer;
+    @BindView(R.id.etBaseUrl)
+    EditText etBaseUrl;
 
     @Inject
     ILoginPresenter presenter;
 
+    private static final String DEFAULT_BASE_URL = "https://eventyay.com/api/v1/";
+
```

Comment:
Actually now we are giving option to change `Constants.BASE_URL` from `LoginActivity`, thats why keeping a DEFAULT url in LoginActivity.